worldedit-core/src/main/java/com/sk89q/worldedit/command/SelectionCommands.java

```diff
@@ -662,78 +671,73 @@ public void select(
-            case POLY: {
+            }
+            case POLY -> {
                 newSelector = new Polygonal2DRegionSelector(oldSelector);
                 actor.print(Caption.of("worldedit.select.poly.message"));
                 Optional<Integer> limit = ActorSelectorLimits.forActor(actor).getPolygonVertexLimit();
                 limit.ifPresent(integer -> actor.print(Caption.of(
                         "worldedit.select.poly.limit-message",
                         TextComponent.of(integer)
                 )));
-                break;
             }
-            case ELLIPSOID:
+            case ELLIPSOID -> {
                 newSelector = new EllipsoidRegionSelector(oldSelector);
                 actor.print(Caption.of("worldedit.select.ellipsoid.message"));
-                break;
-            case SPHERE:
+            }
+            case SPHERE -> {
                 newSelector = new SphereRegionSelector(oldSelector);
                 actor.print(Caption.of("worldedit.select.sphere.message"));
-                break;
-            case CYL:
+            }
+            case CYL -> {
                 newSelector = new CylinderRegionSelector(oldSelector);
                 actor.print(Caption.of("worldedit.select.cyl.message"));
-                break;
-            case CONVEX:
-            case HULL:
-            case POLYHEDRON: {
+            }
+            case CONVEX, HULL, POLYHEDRON -> {
                 newSelector = new ConvexPolyhedralRegionSelector(oldSelector);
                 actor.print(Caption.of("worldedit.select.convex.message"));
                 Optional<Integer> limit = ActorSelectorLimits.forActor(actor).getPolyhedronVertexLimit();
                 limit.ifPresent(integer -> actor.print(Caption.of(
                         "worldedit.select.convex.limit-message",
                         TextComponent.of(integer)
                 )));
-                break;
             }
+
             //FAWE start
-            case POLYHEDRAL:
+            case POLYHEDRAL -> {
                 newSelector = new PolyhedralRegionSelector(world);
                 actor.print(Caption.of("fawe.selection.sel.convex.polyhedral"));
                 Optional<Integer> limit = ActorSelectorLimits.forActor(actor).getPolyhedronVertexLimit();
                 limit.ifPresent(integer -> actor.print(Caption.of("fawe.selection.sel.max", integer)));
                 actor.print(Caption.of("fawe.selection.sel.list"));
-                break;
-            case FUZZY:
-            case MAGIC:
+            }
+            case FUZZY, MAGIC -> {
                 Mask maskOpt = new IdMask(world);
                 newSelector = new FuzzyRegionSelector(actor, world, maskOpt);
                 actor.print(Caption.of("fawe.selection.sel.fuzzy"));
                 actor.print(Caption.of("fawe.selection.sel.list"));
-                break;
+            }
             //FAWE end
-            case LIST:
-            default:
+            case LIST, default -> {
```

Comment:
Patterns in switches are only available as preview in Java 17, not GA yet.

Reply:
Poor IJ setup ig smh